app/classifier/drawing-tools/ellipse.cjsx

```diff
@@ -53,6 +56,13 @@ module.exports = React.createClass
     x: @props.mark.rx * Math.cos theta
     y: -1 * @props.mark.ry * Math.sin theta
 
+  dragEnd: (target) ->
+    @context.geordi?.logEvent type: 'drag-ellipse'
+    deleteIfOutOfBounds target
+
+  logResize: ->
+    @context.geordi?.logEvent type: 'resize-ellipse'
+
```

Comment:
Following the discussion, we have agreed that what we need to do for now is limit it so that these scrollEnd, dragEnd etc events are only logged for the Gravity Spy project ID.  In future we should improve this with a project preference. I will create a separate issue for that.